app/assets/javascripts/directives/int_greater_than_zero.js

```diff
@@ -0,0 +1,22 @@
+ManageIQ.angular.app.directive('intGreaterThanZero', function() {
+  return {
+    require: 'ngModel',
+    link: function(_scope, _elem, _attrs, ctrl) {
+      ctrl.$validators.intGreaterThanZero = function(modelValue, viewValue) {
```

Comment:
We should probably split these directives differently..

Right now, you have..

1. is float && >0
2. is int && >0
3. is int && >= 0

This looks like it could better split as..

1. is int
2. is float
3. is >0
4. is >=0

.. and since we're using `angular.validators` (https://github.com/gkaimakas/angular.validators) ...

1. We already have `is-float`
2. We already have `is-int`
3. `input type=number min=0` already validates >= 0, but the only way to test a float for >0 would be step="0.01" + min="0.01", which may be undesirable .. but we can have a `greaterThanZero` directive..
3. `input type=number min=0` already validates >= 0, unless you can't use an input type number, in which case we'd need a greaterOrZero as well.